scripts/provisioning/_s3_setup

```diff
@@ -0,0 +1,325 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+# TODO: change name of this python script to 's3_setup' when shell script of the same name is EOL'ed
+import sys
+import os
+import shutil
+import argparse
+import traceback
+import inspect
+import errno
+import json
+import ldap
+
+from s3confstore.cortx_s3_confstore import S3CortxConfStore
+from s3cipher.cortx_s3_cipher import CortxS3Cipher
+from cortx.utils.validator.v_pkg import PkgV
+from cortx.utils.validator.v_service import ServiceV
+from cortx.utils.validator.v_path import PathV
+
+"""Constants"""
+LDAP_USER = "cn={},dc=seagate,dc=com"
+LDAP_URL = "ldapi:///"
+
+CONFSTORE_OPENLDAP_CONST_KEY = 'openldap'
+
+CONFSTORE_LDAPADMIN_USER_KEY = 'openldap>sgiam>user'
+CONFSTORE_LDAPADMIN_PASSWD_KEY = 'openldap>sgiam>secret'
+
+#CONFSTORE_ROOTDN_USER_KEY = 'openldap>root>user'
+#CONFSTORE_ROOTDN_PASSWD_KEY = 'openldap>root>secret'
+
+class Cmd:
```

Comment:
can we move these classes to different files, we can keep only orchestration part in this file

Reply:
done